docs/pages/get-started/installation.md

```diff
@@ -4,50 +4,64 @@ title: Installation
 
 ### Installing Expo CLI
 
+Expo CLI is installed as a global npm package from a terminal app on your development machine. You will use it for different tasks in the development life-cycle of your project.
+
 <Terminal cmd={[
   '# Install the command line tools',
   '$ npm install --global expo-cli'
 ]} />
 
-Verify that the installation was successful by running `expo whoami`. You're not logged in yet, so you will see "Not logged in". You can create an account by running `expo register` if you like, or if you have one already run `expo login`, but you also don't need an account to get started.
+To verify the successful installation of CLI, run the following command:
+
+<Terminal cmd={['$ expo whoami']} />
+
+If the installation is successful, you will see a "Not logged in" message since you are not logged in to an Expo account yet. You do not need an account to start and can proceed further with your project. However, if you want to register a new expo account, run the command:
+
+<Terminal cmd={['$ expo register']} />
+
+If you already have an Expo account, you can log into it by running the command:
+
+<Terminal cmd={['$ expo login']} />
 
-> 😳 **Need help?** Try searching the [forums](https://forums.expo.dev) &mdash; which are a great resource for troubleshooting.
+> **Need help?** Try searching the [forums](https://forums.expo.dev) &mdash; which is a great resource for troubleshooting.
 
 ## 2. Expo Go app for iOS and Android
 
-The fastest way to get up and running is to use the Expo Go app on your iOS or Android device. Expo Go allows you to open up apps that are being served through Expo CLI.
+The fastest way to get up and running is to use the Expo Go app on your iOS or Android device. Expo Go allows you to open up apps served through Expo CLI. It is available on both the iOS App Store and Android Play Store.
 
-- 🤖 [Android Play Store](https://play.google.com/store/apps/details?id=host.exp.exponent) - Android Lollipop (5) and greater.
-- 🍎 [iOS App Store](https://apps.apple.com/app/expo-go/id982107779) - iOS 11 and greater.
+- [Android Play Store](https://play.google.com/store/apps/details?id=host.exp.exponent) - Android Lollipop (5) and greater.
+- [iOS App Store](https://apps.apple.com/app/expo-go/id982107779) - iOS 11 and greater.
 
-When the Expo Go app is finished installing, open it up. If you created an account with `expo-cli` then you can sign in here on the "Profile" tab. This will make it easier for you to open projects in the client when you have them open in development &mdash; they will appear automatically in the "Projects" tab of the client app.
+Open the Expo Go app after it has finished installing. If you have created an account with `expo-cli`, you can sign in within the app from the "Profile" tab. Signing in will make it easier for you to open projects in the Expo Go app while developing them &mdash; they will appear automatically in the "Projects" tab of the app.
```

Comment:
```suggestion
Open the Expo Go app after it has finished installing. If you have created an account with `expo-cli`, you can sign in from the "Profile" tab. Signing in will make it easier for you to open projects in the Expo Go app while developing them &mdash; they will appear automatically in the "Projects" tab of the app.
```
Sentence starts with 'open the app', so the profile tab is already implied to be inside the app.